defaults/main.yml

```diff
@@ -45,6 +45,10 @@ dataverse:
     upload_to_hub: False
     user: counter
     year_month: "2018-05"
+  custom_metadata_blocks:
+    enabled: true
+    urls:
+      - https://github.com/IQSS/dataverse/files/3744336/codemeta.tsv.txt
```

Comment:
If it's disabled by default it's probably fine to leave this URL here. People will figure out to swap in the URLs they want.